tests/TESTS/LinkLoss/host/test_link_loss.py

```diff
@@ -83,10 +78,6 @@ async def test_alert_mechanism(board, client, alert_level, alert_message):
 async def test_disconnection_reconnection(board, client, alert_level, alert_message):
     await client.write_gatt_char(UUID_ALERT_LEVEL_CHAR, alert_level)
     await client.write_gatt_char(UUID_DISCONNECTION_REASON_CHAR, CONNECTION_TIMEOUT)
-    # On Windows, we need to trigger a normal disconnection to prevent Bleak from trying to automatically reconnect
-    # This does not affect the application running on the device since we are already disconnected
-    if platform.system() == "Windows":
-        await client.disconnect()
-    board.wait_for_output(alert_message, timeout=10)
+    await board.wait_for_output(alert_message, timeout=10)
```

Comment:
this doesn't mean bleak disconnected, it just means the board disconnected so bleak might still think it's connected

Reply:
Surprisingly, the issue appears to be with latest bleak [0.11.0](https://pypi.org/project/bleak/0.11.0/).

Rolling back to [0.10.0](https://pypi.org/project/bleak/0.10.0/) seems to solve it: 63c33637cc9d2e02bef4208400af588373db1165

Reply:
Actually, it never worked. Same errors on github-ci. Only passes due to timeout.